ISM43362/ATParser/BufferedSpi/BufferedSpi.h

```diff
@@ -191,14 +191,11 @@ class BufferedSpi : public SPI
     /**
     * Allows timeout to be changed between commands
     *
-    * @param timeout timeout of the connection
+    * @param timeout timeout of the connection in ms
     */
     void setTimeout(int timeout)
     {
-        /*  this is a safe guard timeout in case module is stuck
-         *  so take 5 sec margin compared to module timeout, to
-         *  really only detect case where module is stuck */
-        _timeout = timeout + 5000;
+        _timeout = timeout;
```

Comment:
Can you clarify  in a comment ? 
When will this function be called now, at init only I guess
And the value would be a large one like 60 sec right ?

Reply:
Explain this will be the transport timeout (SPI level, in case no answer)